base/setup/lib/utils/partlist.c

```diff
@@ -2720,7 +2774,7 @@ UpdateDiskLayout(
  *
  * @return  The adjacent unpartitioned region, if it exists, or NULL.
  **/
-static
+//static
```

Comment:
Is this function used from the outside now? If yes, strip the line entirely, if not, remove the `//` instead.

Reply:
It is temporarily used from outside, because of a hack in the gui setup (in drivepage.c) that I kept here to make this PR self-contained and compile. I plan to merge PR #7162 first and then rebase the current PR: this will remove the need for this hack in the final version.

Reply:
I've just rechecked this commit, and actually I use it in more than one place now, so I'll clean this now.